.github/workflows/runtime-tests.yml

```diff
@@ -25,6 +28,14 @@ jobs:
         uses: actions/checkout@v4
         with:
           submodules: true
+      - name: Setup dotnet
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: ${{env.dotnet-version}}
```

Comment:
Definitely delete the first copy (and the useless `env`)!

Reply:
What do you mean? I'm purposefully installing both 8 and 6, so I'm invoking `actions/setup-dotnet@v4` twice